src/main/webapp/styles/recmap.css

```diff
@@ -0,0 +1,65 @@
+/* ----- Recommendation Map CSS ----- */
```

Comment:
same here, please use `recommendation` instead of `rec` in the file name and separate both words with a dash :)